helm-charts/helm-repo/templates/deployment.yaml

```diff
@@ -53,7 +53,7 @@ spec:
       tolerations:
 {{ toYaml .Values.tolerations | indent 8 }}
       containers:
-      {{ if .Values.auditService.config.enabled }}
+      {{ if eq .Values.auditService.config.enabled "true" }}
```

Comment:
I think `"true"` shouldn't be surrounded in quotes. Seems to be causing a helm linting error that's affecting PROW